recipes/samarium/all/CMakeLists.txt

```diff
@@ -1,7 +1,4 @@
 cmake_minimum_required(VERSION 3.15)
```

Comment:
If you are using CMakeToolchain, this cmake wrapper is no longer needed, it can be removed.